app/Http/Requests/PostRequest.php

```diff
@@ -23,8 +23,8 @@ public function rules()
     {
         return [
             'northstar_id' => 'required|string',
-            'campaign_id' => 'required|int',
-            'campaign_run_id' => 'required|int',
+            'campaign_id' => 'required',
+            'campaign_run_id' => 'int',
```

Comment:
Do you think it's worth it to add [Complex Conditional Validation](https://laravel.com/docs/5.5/validation#conditionally-adding-rules) here to make `campaign_run_id` required when the `campaign_id` is an integer or if that's something we should look at handling elsewhere?

Reply:
@katiecrane sorry i missed this! Since we actually want to deprecate `campaign_run_id` entirely so i'm not too concerned with adding a bunch of validation around it.

Reply:
@sbsmith86 cool!!!! 🔥 🏃